### ISSUE TITLE
text format supports variants, but render only kinda does

### ISSUE THREAD

**Issue by Yamavu** [IdreesInc/PlayBook#8]
I was wondering why weird font changes were happening, didn't find anything in code and researched a bit.
turns out Playbook supports bold and italic via the standard library 

`playdate.graphics.drawText("normal *bold* _italic_", x, y)`

- `*bold*` renders as bold text
- `_italic_` renders as italic text

![PlayBook 2024-03-20 08 17 39](https://github.com/IdreesInc/PlayBook/assets/4389401/c310d045-2dd0-476d-a1d2-2e9a2bfc0265)

for that to work, the bold and italic variants have to be set alongside the regular font. (see [Inside Playdate](https://sdk.play.date/2.4.2/Inside%20Playdate.html#f-graphics.setFont))

(On the other hand if this behavior is unwanted, [`playdate.graphics.font:drawText()`](https://sdk.play.date/2.4.2/Inside%20Playdate.html#m-graphics.font.drawText) instead of `playdate.graphics.drawText()`. I think it makes this app more usable and loosens up the text.)

**Comment by IdreesInc** [comment]
Oh, that's unexpected! Didn't expect that, maybe text formatting is more possible than I thought, will have to keep this in mind if I add the epub to txt converter in the future. Also I see that you are using possibly the very first version of PlayBook, heads up that the updated versions are available on itch.io (if you downloaded it there previously) or on the catalog!

**Comment by Yamavu** [comment]
yeah I'll have to wrap my head around the way Playdata does fonts and actually make em, I'd have a Pull Request ready pretty soon. 
The silly idea that started this was to get a flavor of [OpenDyslexic](https://en.wikipedia.org/wiki/OpenDyslexic) to work on that little screen. 
Thanks for the headsup. I didn't find the itchio page yet
